### DISCUSSION TITLE
Hundreds of these messages every minute across tons of different ports and both my network adapters (wired + wireless). What's going on?

### DISCUSSION THREAD

**Discussion by kingkj52** [AdguardTeam/AdGuardHome#7393] [Q&A]
This message is getting spammed a lot. Am I getting port scanned or is it a sign of a virus or something?

2024/10/31 20:09:43.882567 [error] dnsproxy: responding request proto=udp err="writing message: write udp [::]:53->192.168.1.100:64034: wsasendto: A message sent on a datagram socket was larger than the internal message buffer or some other network limit, or the buffer used to receive a datagram into was smaller than the datagram itself."

The port changes every message, and the IP bounces between my wired and wireless local network IP every message, hundreds per minute.

**Comment by Dynamic5912** [comment]
How is this related to AGH?

**Reply by kingkj52** [reply]
These are popping up in AdGuard Home's terminal and its causing service degradation and higher resource usage for AGH that is not happening while it is not running. Hence I am asking if this is a personal thing (port scan, virus, what have you) that is unrelated to AGH, and AGH just happens to be on the receiving end of it due to the nature of what it does, or if this is a bug that I should be looking to fix.

Also asking for more info on what the message itself means, I tried looking into it and didn't find much, and I admit networking is basically magic to me. I tagged it as Q&A and not as an issue, so I'd say it has a lot to do with AGH and is in the right place.

**Reply by Dynamic5912** [reply]
Is your AGH instance public facing?
